src/atoms/Text/Text.native.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import theme from '../../tokens/theme';
+import { Text as NativeText } from 'react-native';
+import { flattenArray } from '../../utils/index';
+
+const fontFamilies = {
+  light: `Lato-Regular`,
+  regular: `Lato-Regular`,
+  bold: `Lato-Bold`,
+};
+
+const Text = styled(NativeText)(
+  (props) => `
+  font-family: ${fontFamilies[props.weight]};
+  font-size: ${props.theme.fonts.size[props.size]};
+  letter-spacing: ${props.theme.fonts.letterSpacing[props.letterSpacing]};
+  color: ${props.color || props.theme.colors.shade[800]};
```

Comment:
As per the figma file the color is `tone` and not `shade`? 🤔

Reply:
I spoke to Nishant regarding this. He will be removing the color column from there. Color for the text will be defined by the component consuming it. I've kept shade only as a fallback.